zap/src/main/java/org/zaproxy/zap/utils/ZapTextField.java

```diff
@@ -129,4 +130,13 @@ public void setEditsLimit(int limit) {
     public void setUndoManagerPolicy(UndoManagerPolicy policy) throws NullPointerException {
         undoManager.setUndoManagerPolicy(policy);
     }
+
+    /**
+     * Sets the placeholder of this {@code ZapTextField}.
+     *
+     * @param labelText the new placeHolder
+     */
+    public void setPrompt(String labelText) {
```

Comment:
Be good to be consistent with the names, I don't mind either `placeholder` or `prompt` (though I'd go with the latter) but we should be consistent everywhere (e.g. JavaDoc, variables, resources keys).